### PR TITLE
Change e2e integration flavor

### PR DESCRIPTION
**Pull request by mboukhalfa**

### CODE REVIEW

File: jenkins/scripts/integration_test.sh [hunk 1]
```diff
@@ -68,7 +68,7 @@ if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${TESTS_FOR}" == "e2e_tests"*
 fi
 echo "Running in region: ${OS_REGION_NAME}"
 
-if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* ]]; then
+if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* && "${GINKGO_FOCUS}" != "integration" ]]; then
```
[the remark this answers is not in the document]
Comment (mboukhalfa) [reply jenkins/scripts/integration_test.sh:71]: we have also upgrade and .. k8s upgrade ..

[the remark this answers is not in the document]
Comment (kashifest) [reply jenkins/scripts/integration_test.sh:71]: ow and all of those need 8C-32GB? Why would controller upgrade need that ?

[the remark this answers is not in the document]
Comment (mboukhalfa) [reply jenkins/scripts/integration_test.sh:71]: upgrade uses 5 VMs btw

**Review by kashifest — APPROVED** [review]

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Pull request merged**